RosettaImmutables/pom.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.hubspot.rosetta</groupId>
+    <artifactId>Rosetta</artifactId>
+    <version>3.11.11-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>RosettaImmutables</artifactId>
+
+  <properties>
+    <hubspot.module.type>CORE</hubspot.module.type>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.hubspot.immutables</groupId>
+        <artifactId>hubspot-style</artifactId>
+        <version>1.3-SNAPSHOT</version>
+        <!-- TODO - do we actually want to do this? -->
```

Comment:
What's the value in this exclusion?

Reply:
It was one way to work around:
```
[ERROR] Found a problem with the direct dependency com.google.guava:guava of the current project
  Expected version is 22.0
  Resolved version is 22.0
  Version 25.0-jre was expected by artifact: com.hubspot.immutables:hubspot-style
[ERROR] Found a problem with the dependency com.google.errorprone:error_prone_annotations
  Resolved version is 2.0.18
  Version 2.0.18 was expected by artifact: com.google.guava:guava
  Version 2.1.3 was expected by artifact: com.hubspot.immutables:hubspot-style
  ```
  Without having to add a dependency management section (and then having to remember to update it)